pom.xml

```diff
@@ -274,7 +274,13 @@
                         <exclude>**/hbase/data/**</exclude>
                         <exclude>**/kafkazk/data/**</exclude>
                         <exclude>**/wait-for-it.sh</exclude>
-			<exclude>**/*.out</exclude>
+			                  <exclude>**/*.out</exclude>
+                        <exclude>node_modules/**</exclude>
```

Comment:
Can you please put comments on each of these exclusions indicating what they are, where they come from and why the format cannot take comments.  We've been ding'd about overbroad exclusions before.